runtime/dev/src/lib.rs

```diff
@@ -928,6 +932,32 @@ parameter_types! {
 	pub XcmWeight: XcmBaseWeight = XCM_WEIGHT.into();
 	pub ContributionWeight:u64 = XCM_WEIGHT.into();
 	pub WithdrawWeight:u64 = XCM_WEIGHT.into();
+	pub ConfirmMuitiSigAccount: AccountId = hex![
```

Comment:
Use `Multisig::multi_account_id()` instead and introduce `pallet-multisig` by the way.